src/desktop_pixmap.cpp

```diff
@@ -31,7 +33,8 @@ desktop_pixmap::desktop_pixmap (
 
   // Create a pixmap
   // TODO This is suboptimal for compressed screenshots
-  create_pixmap ();
+  create_pixmap (800,
+                 (1080 * (800 / 1920 + 1))); // resized width, resized height;
```

Comment:
Elaborate on this calculations in the comment please

Reply:
Written above